apps/cookbook/src/app/examples/accordion-example/examples/card-list.ts

```diff
@@ -0,0 +1,97 @@
+import { Component } from '@angular/core';
+
+const config = {
+  selector: 'cookbook-accordion-with-list-in-card-example',
+  template: `<kirby-card>
+  <kirby-accordion>
+    <kirby-accordion-item title="Transactions">
+      <kirby-list [items]="transactions.slice(0,4)">
+        <kirby-item *kirbyListItemTemplate="let item">
+          <kirby-label>
+            <h3>{{item.title}}</h3>
+            <p subtitle>{{item.subTitle}}</p>
+          </kirby-label>
+          <kirby-label slot="end">
+            <data [value]="item.amount">{{item.amount}}</data>
+            <data [value]="item.detail" detail>{{item.detail}}</data>
+          </kirby-label>
+        </kirby-item>
+      </kirby-list>
+    </kirby-accordion-item>
+    <kirby-accordion-item title="More Transactions">
+      <kirby-list [items]="transactions.slice(4)">
+        <kirby-item *kirbyListItemTemplate="let item">
+          <kirby-label>
+            <h3>{{item.title}}</h3>
+            <p subtitle>{{item.subTitle}}</p>
+          </kirby-label>
+          <kirby-label slot="end">
+            <data [value]="item.amount">{{item.amount}}</data>
+            <data [value]="item.detail" detail>{{item.detail}}</data>
+          </kirby-label>
+        </kirby-item>
+      </kirby-list>
+    </kirby-accordion-item>
+  </kirby-accordion>
+</kirby-card>`,
+};
+
+@Component({
+  selector: config.selector,
+  template: config.template,
+})
+export class AccordionWithItemsInCardExampleComponent {
```

Comment:
You can extend the `BaseListComponent` here, to avoid duplicating the transactions array 🤘